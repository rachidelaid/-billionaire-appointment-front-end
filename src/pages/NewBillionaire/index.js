import React from 'react';

const NewBillionaire = () => (
  <div>
    <Navbar />
  </div>
);

export default NewBillionaire;
