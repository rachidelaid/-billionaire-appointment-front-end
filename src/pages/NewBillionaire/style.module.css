.d-flex {
  display: flex;
}

.col {
  flex-direction: column;
}

.alert-ctn {
  color: red;
}

.click {
  cursor: pointer;
}
