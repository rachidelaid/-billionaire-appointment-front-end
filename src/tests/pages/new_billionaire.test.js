import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Wrapper from '../helpers/test_wrapper';
import NewBillionaire from '../../pages/NewBillionaire';

describe('New Billionaire Page', () => {
  const page = <Wrapper><NewBillionaire /></Wrapper>;
  const rendered = render(page);

  it('should render the Add a New Billionaire page', async () => {
    expect(rendered).toMatchSnapshot();
  });

  it('should display the title', async () => {
    render(page);
    expect(screen.queryByText(/ADD A BILLIONAIRE/i)).toBeInTheDocument();
  });
});
