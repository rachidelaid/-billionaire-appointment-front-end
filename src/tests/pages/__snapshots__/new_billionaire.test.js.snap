// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Billionaire Page should render the Add a New Billionaire page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="section-ctn d-flex col"
      >
        <div
          class="layer d-flex col"
        >
          <div
            class="form-parent d-flex col"
          >
            <h2>
              ADD A BILLIONAIRE
            </h2>
            <hr />
            <div
              class="form-ctn d-flex col"
            >
              <div>
                Please Log In before adding billionaires
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="section-ctn d-flex col"
    >
      <div
        class="layer d-flex col"
      >
        <div
          class="form-parent d-flex col"
        >
          <h2>
            ADD A BILLIONAIRE
          </h2>
          <hr />
          <div
            class="form-ctn d-flex col"
          >
            <div>
              Please Log In before adding billionaires
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
